### PR TITLE
Moving to Ubuntu Bionic on Travis

### PR DESCRIPTION
**Pull request by lekshmideepu**
Running CI builds on Ubuntu Bionic. Using more recent versions of some of the libraries like Boost 1.65.1, removed the python2 packages from the source list.

### CODE REVIEW

**Comment by lekshmideepu** [comment]
Fixes #1542

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3605514.svg)](https://doi.org/10.5281/zenodo.3605514)
 [![Latest Version](https://img.shields.io/badge/latest%20version-2.20.0-brightgreen.svg)](https://github.com/nest/nest-simulator/releases/tag/v2.20.0)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2218/badge)](https://bestpractices.coreinfrastructure.org/projects/2218)
+[![Coverage Status](https://coveralls.io/repos/github/lekshmideepu/nest-simulator/badge.svg?branch=master)](https://coveralls.io/github/lekshmideepu/nest-simulator?branch=master)
```
Comment (jougs) [inline README.md:8]: Did this line go in by accident or is coveralls already enabled and I did not notice?

Comment (lekshmideepu) [reply README.md]: Probably by accident. I had enabled coveralls as part of the testing process. I just fixed the file. Thanks for the review

**Comment by lekshmideepu** [comment]
> Thanks for this contribution. 👍 Approved!

Thanks @gtrensch  for the review!

**Pull request merged**